app/models/ability.rb

```diff
@@ -8,6 +8,10 @@ class Ability
 
   # Define any customized permissions here.
   def custom_permissions
+    if current_user.admin?
+      can [:create, :show, :add_user, :remove_user, :index, :edit, :update, :destroy], Role
```

Comment:
Should this check against the user group and not just their abilities?  There's this in CC too: https://github.com/projecthydra/curation_concerns/blob/master/app/models/concerns/curation_concerns/ability.rb#L40